app/assets/v2/js/grants/new.js

```diff
@@ -82,26 +82,31 @@ const init = () => {
             $('#transaction_hash').val(transactionHash);
             const linkURL = etherscan_tx_url(transactionHash);
 
-            let data = {
-              'title': $('#input_title').val(),
-              'description': $('#input-description').val(),
-              'reference_url': $('#input-url').val(),
-              'admin_address': $('#input-admin_address').val(),
-              'contract_owner_address': $('#contract_owner_address').val(),
-              'token_address': $('#token_address').val(),
-              'token_symbol': $('#token_symbol').val(),
-              'amount_goal': $('#amount_goal').val(),
-              'contract_version': $('#contract_version').val(),
-              'transaction_hash': $('#transaction_hash').val(),
-              'network': $('#network').val(),
-              'team_members': $('#input-team_members').val(),
-              'csrfmiddlewaretoken': $("#create-grant input[name='csrfmiddlewaretoken']").val()
-            };
+            let file = $('#img-project')[0].files[0];
+            let formData = new FormData();
```

Comment:
Since we're using `FormData()` here, why not skip all the append calls below and use `FormData(document.getElementById('create-grant'))`?

Reply:
If we are doing that , we'd have to update the html ids to match like `input_team_members` now become `team_members`

Reply:
@danlipert The main reason is that the first POST call will then contain all input id's. Currently, we use the id's being sent through as a way to differentiate between the first (saving most data) and second (saving contract_address and redirecting page) POST calls. This way we can perform different actions after saving data in grants/views.py. If there is a better way to achieve this functionality, I am all for best practices. This was just the solution I came to. Let me know what you think!